### PR TITLE
Extend Layers Control tutorial to add layer dynamically

### LINKED ISSUE #7787: Extend `Layers Control` Tutorial with dynamic change of the layers
I suggest to add a section in the [Layers Control tutorial](https://leafletjs.com/examples/layers-control/) where is described how can layers added and removed dynamically.

Docu: https://leafletjs.com/reference.html#control-layers-addbaselayer

PR is welcome.

### PR DESCRIPTION
**Pull request by QuentinPhilipp**
Hello! 

This PR resolves #7787 by adding an example how to add a base layer and an overlay dynamically in a Layers Control. 

I've also updated the code for the live example.

### CODE REVIEW

File: docs/SlavaUkraini/examples/layers-control/index.md [hunk 1]
```diff
@@ -54,19 +54,32 @@ var overlayMaps = {
...
 <pre><code>var baseMaps = {
 	"&lt;span style='color: gray'&gt;Grayscale&lt;/span&gt;": grayscale,
 	"Streets": streets
 };
 </code></pre>
 
+Finally, you can add or remove base layers or overlays dynamically:
+
+<pre><code>var crownHill = L.marker([39.75, -105.09]).bindPopup('This is Crown Hill Park.'),
+    rubyHill = L.marker([39.68, -105.00]).bindPopup('This is Ruby Hill Park.')
```
Comment (Falke-Design) [inline docs/SlavaUkraini/examples/layers-control/index.md:74]:
```suggestion
    rubyHill = L.marker([39.68, -105.00]).bindPopup('This is Ruby Hill Park.');
```

File: docs/SlavaUkraini/examples/layers-control/example.md [hunk 2]
```diff
@@ -32,4 +32,11 @@ title: Layers Control Tutorial
 	};
 
 	var layerControl = L.control.layers(baseLayers, overlays).addTo(map);
+	var crownHill = L.marker([39.75, -105.09]).bindPopup('This is Crown Hill Park.'),
+	    rubyHill = L.marker([39.68, -105.00]).bindPopup('This is Ruby Hill Park.')
```
Comment (Falke-Design) [inline docs/SlavaUkraini/examples/layers-control/example.md:36]:
```suggestion
	    rubyHill = L.marker([39.68, -105.00]).bindPopup('This is Ruby Hill Park.');

```

**Review by Falke-Design — CHANGES_REQUESTED** [review]
Please add missing semicolon, else it looks good. 
grafik

**Comment by QuentinPhilipp** [comment]
I approved proposed fix

**Comment by QuentinPhilipp** [comment]
@Falke-Design I've noticed an issue in the code, the provided example I gave couldn't work since the name of the control layer wasn't correct. I fixed it and updated the thumbnail to match the new example as well.

**Review by Falke-Design — CHANGES_REQUESTED** [review]
Please "zoom into" the picture, like before and the width should be 750px

grafik

**Comment by QuentinPhilipp** [comment]
I changed the size and zoomed into the picture. I don't know why the grayscale tile isn't rendering exactly the same as the previous example, I used the same map ID, maybe the map theme changed a little from the last time this example was made.

**Review by Falke-Design — APPROVED** [review]
Very nice. Thank you for your contribution!

**Comment by QuentinPhilipp** [comment]
The thumbnail has been compressed from 157KB to 84KB

**Review by Falke-Design — APPROVED** [review]

**Review by Malvoz — APPROVED** [review]

**Comment by Falke-Design** [comment]
Thank you @QuentinPhilipp

**Pull request merged**